storage_server/report_server.py

```diff
@@ -239,14 +230,14 @@ def needFileContent(self, run_id, filepath):
         f = self.session.query(File) \
                         .filter(and_(File.run_id == run_id,
                                      File.filepath == filepath)) \
-                        .first()
+                        .one_or_none()
```

Comment:
Please update the minimal required SQLAlchemy version in the documentation too to a higher one if a new query is used which is only available from 1.0.9 and handle the possible sqlalchemy.orm.exc.MultipleResultsFound exception.